Dev

Denisyasyuchenya opened the pull request:
MOV-1 is ready to check out. I have some problems with FOREIGN KEY (it dosn't work in my script). So can you tell me how to solve this problem? Thx, and sorry about delaying the tasks. I was busy because my relatives came in Minsk from Chelabinsk and I was forced to spend my time with them) But now I have more spare time.

Sergey113222 committed:
Only one table
1.Create
2.Config DataSource
3.Repository
4.Create controller

Denisyasyuchenya committed:
mov-1

RomanKashenok commented on src/main/java/info/movietrash/cinemabase/controller/UserController.java:
```diff
@@ -0,0 +1,45 @@
+@RestController
+@RequestMapping(value = "/movietrash/api/users")
+public class UserController {
+    private final UserRepository userRepository;
```
Не, давай репозитории не будем сетить в контроллер. Положи их в сервис

RomanKashenok commented on src/main/java/info/movietrash/cinemabase/controller/UserController.java:
```diff
@@ -0,0 +1,45 @@
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/movietrash/api/users")
```
/api/users instead of /movietrash/api/users

RomanKashenok commented on src/main/java/info/movietrash/cinemabase/controller/UserController.java:
```diff
@@ -0,0 +1,45 @@
+
+    @DeleteMapping("/{id}")
+    public void deleteUserById(@PathVariable("id") Long id) {
+        userRepository.deleteById(id);
```
мы не будем удалять пользователей. Мы будем их помечать как active = false

RomanKashenok commented on src/main/java/info/movietrash/cinemabase/model/Status.java:
```diff
@@ -0,0 +1,5 @@
+package info.movietrash.cinemabase.model;
+
+public enum Status {
```
не, давай уберем этот энам. он бесполезен

RomanKashenok commented on src/main/java/info/movietrash/cinemabase/model/User.java:
```diff
@@ -0,0 +1,43 @@
+    private Role role;
+    @Column(name = "status")
+    @Enumerated(EnumType.STRING)
+    private Status status;
```
убираем статус

Denisyasyuchenya committed:
mov-1

Denisyasyuchenya committed:
mov-1

Commit:
MOV-1: refactoring

Sergey113222 committed:
1. Roma's update DB
2. Add apikey.properties

Sergey113222 committed:
1. Roma's update DB
2. Add apikey.properties

Sergey113222 committed:
1. add UserDto
2. add UserConverter
3. add UserService & UserServiceImpl
4. change controller(work with service, not repository)
